pyomo/core/tests/unit/test_numeric_expr.py

```diff
@@ -2591,7 +2591,7 @@ def test_invalidIndexing(self):
         #
         # Check for errors when generating expressions with invalid indices
         #
-        m = AbstractModel()
+        m = ConcreteModel()
```

Comment:
Why change from `Abstract` to `Concrete` here?

Reply:
It was necessary during development when I was close to getting matrix opreations between indexed Pyomo objects working.  However, that feature ended up being pulled from this PR due to other challenges.  I have reverted the change here.